pkg/artifacts/signable.go

```diff
@@ -197,6 +216,7 @@ func ExtractOCIImagesFromResults(ctx context.Context, obj objects.TektonObject)
 	logger := logging.FromContext(ctx)
 	objs := []interface{}{}
 
+	logger.Infof("aprindle-10 - here")
```

Comment:
I guess this was for debugging. Can be removed.

Reply:
done